README.md

```diff
@@ -1,7 +1,331 @@
 # Coveo C# Platform SDK
-The offical Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+[![NuGet version (Coveo.Connectors.Utilities.PlatformSdk)](https://img.shields.io/nuget/v/Coveo.Connectors.Utilities.PlatformSdk.svg?style=flat-square)](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/)
 
-The code of the SDK is not open-source. This repo is meant to provide feedback, bug reporting, etc.
+The official Coveo C# SDK allows your software to communicate with the Coveo Cloud Platform public APIs and the Push API.
+
+The package is .NET Standard 2.0 compatible and therefore works with .NET Core and .NET Framework
+
+The code of the SDK is not open-source. This repository is meant to provide feedback, bug reporting, etc.
+
+## Installation
+Install the public NuGet package [Coveo.Connectors.Utilities.PlatformSdk](https://www.nuget.org/packages/Coveo.Connectors.Utilities.PlatformSdk/) in your C# project using your favorite IDE or NuGet command line.
+
+The package is signed before being published to NuGet to avoid pulling a malicious package.
+We provide the `PDBs` to have a better debugging experience.
+Documentation is available for most of the public members and methods.
 
 ## How to use the SDK
-Coming soon!
+First, you need to instantiate the client to interact with the platform. Here is the minimum configuration you need to provide:
+
+
```

Comment:
** Each section below will redirect you to the privileges needed for the requests to work. For more information about API key privileges, visit [Privilege Reference](https://docs.coveo.com/en/1707/cloud-v2-administrators/privilege-reference).**